lib/modules/datasource/glasskube-packages/__fixtures__/versions.yaml

```diff
@@ -0,0 +1,4 @@
+latestVersion: v1.23.1+1
```

Comment:
is this always the highest version? if not, then we should return it as latest tag, so users can use follow tag config option.

Reply:
No, as previously discussed (https://github.com/renovatebot/renovate/pull/29430#discussion_r1626325139), there might be higher versions than this.

Reply:
add it as tags.latest on release result. so users can opt-in to follow that tag to only allow version lower or equal to this version.